setools/policyrep/terule.pxi

```diff
@@ -536,8 +544,15 @@ cdef class FileNameTERule(BaseTERule):
             yield self
 
     def statement(self):
-        return "{0.ruletype} {0.source} {0.target}:{0.tclass} {0.default} {0.filename};". \
-            format(self)
+        name_match_str = "???"
+        if self.name_match == sepol.NAME_TRANS_MATCH_EXACT:
+            name_match_str = "";
+        if self.name_match == sepol.NAME_TRANS_MATCH_PREFIX:
```

Comment:
Seems that this should be an `elif`.

Reply:
Yes, you're right. I have fixed it now, thank you.